tests/jballerina-unit-test/src/test/resources/test-src/reachability-analysis/unreachability_test.bal

```diff
@@ -1770,3 +1770,10 @@ class IterableWithError {
         };
     }
 }
+
+function testUnreachableCodeWithWhile7() returns int {
+    int x = 0;
+    while true {
```

Comment:
Shall we add a test with a constant variable and it defined as `true` and use it here.

And also lets add few tests with break, check statements to check the functionality is correct.